src/styles/partials/components/_table.scss

```diff
@@ -0,0 +1,88 @@
+/* BASIC TABLE STYLES */
+.table {
+  border: 1px solid $gray-light;
+  border-collapse: collapse;
+  color: #4c4c4c;
+  font-family: $default-heading-font;
+  font-size: 16px;
+  text-align: left;
+  width: 100%;
+  td {
+    border-bottom: 1px solid $gray-light;
+    padding: 15px;
+  }
+  thead {
+    th {
+      background-color: $white;
+      border-bottom: 5px solid $blue;
+      border-left: none;
+      color: $black;
+      font-family: $default-heading-font;
+      font-size: 18px;
+      font-weight: bold;
+      padding: 15px;
+    }
+  }
+  th {
+    border-bottom: 1px solid $gray-light;
+    border-left: 5px solid $blue;
+    color: $black;
+    padding-left: 20px;
+  }
+  tbody {
+    tr {
+      &:nth-of-type(2n + 1) {
+        background-color: $gray-lightest;
+      }
+      &:hover {
+        background-color: $gold-lightest;
+      }
+    }
+  }
+}
+
+/* BLUE TABLE STYLES */
+.dg_section.dark {
```

Comment:
this logic should move to the sections file and should not move over any properties are duplicated in the default table.

Reply:
@theilman7 I actually thought about this a little this weekend. And im torn of whether the dark sections css should all go in teh sections file or if they should go in the component file. Do you have any thoughts?

Reply:
@martypowell I'm leaning towards the component file. I feel like keeping the dark section styes in the component file will help tidy the structure, since the dark section version of the component, is essentially just a different variation of the base component in relation to the overall taxonomy. Looking ahead, it seems more intuitive that the dark section version of the component would be listed in the main component file, than someone fresh, coming in and heading straight for the component file, not seeing the dark section version and having to then find the section file. In addition, I feel like adding every dark section component style to the section file would create quite a large and muddied file after awhile.

Reply:
i think this is a good point. Here is the argument against that approach

- Say we add a `.light` section. In that case, we would have to go into each component (and know that component has styles for that section, which wouldn't be horrible, becasue we could look for `.dg_section.dark`)
- If they are separated into a section file, then we can copy and paste the existing dark section and make the changes as needed
- Say we need to remove the dark section, then we have to touch all these different file parts instead of one location.

Do these points impact your thoughts at all?